packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -286,6 +286,7 @@ class Dropdown extends React.Component {
                                 icon={this.props.is_alignment_left ? 'IcChevronLeft' : 'IcChevronDown'}
                                 className={classNames('dc-dropdown__select-arrow', {
                                     'dc-dropdown__select-arrow--left': this.props.is_alignment_left,
```

Comment:
unrelated to this, but looks like `--left` is not used in `scss`, can you check?

Reply:
Maybe some considerations to support another type of drop-down.
@negar-binary what do you think?

Reply:
@reza-fs it is used, what do you mean? line 190 dropdown.scss